editor.planx.uk/src/pages/FlowEditor/components/forms/FormModal.tsx

```diff
@@ -83,7 +83,7 @@ const NodeTypeSelect: React.FC<{
         <option value={TYPES.Pay}>Pay</option>
       </optgroup>
       <optgroup label="Outputs">
-        <option value={TYPES.Send}>Send (Prototype)</option>
+        <option value={TYPES.Send}>Send</option>
```

Comment:
🎉

Reply:
thank you for noticing this very minor but momentous change lol :tada: